sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/Constants.java

```diff
@@ -2,29 +2,29 @@
 // Licensed under the MIT License.
 package com.azure.spring.data.cosmos;
 
-import com.azure.data.cosmos.IndexingMode;
+import com.azure.cosmos.models.IndexingMode;
 
 /**
  * Constants class of CosmosDB properties
  */
 public final class Constants {
 
-    public static final String DEFAULT_COLLECTION_NAME = "";
-    public static final boolean DEFAULT_INDEXINGPOLICY_AUTOMATIC = true;
-    public static final IndexingMode DEFAULT_INDEXINGPOLICY_MODE = IndexingMode.CONSISTENT;
+    public static final String DEFAULT_CONTAINER_NAME = "";
+    public static final boolean DEFAULT_INDEXING_POLICY_AUTOMATIC = true;
+    public static final IndexingMode DEFAULT_INDEXING_POLICY_MODE = IndexingMode.CONSISTENT;
     public static final String DEFAULT_REPOSITORY_IMPLEMENT_POSTFIX = "Impl";
     public static final int DEFAULT_TIME_TO_LIVE = -1; // Indicates never expire
     public static final boolean DEFAULT_AUTO_CREATE_CONTAINER = true;
 
     public static final String ID_PROPERTY_NAME = "id";
 
-    public static final String COSMOSDB_MODULE_NAME = "cosmosdb";
-    public static final String COSMOSDB_MODULE_PREFIX = "cosmosdb";
+    public static final String COSMOS_MODULE_NAME = "cosmosDb";
```

Comment:
I think the brand name is Cosmos not CosmosDB.

Reply:
Yes, I am pushing an update for these. thanks.